gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/mapper/KeyMapper.java

```diff
@@ -51,6 +55,14 @@ public Key convert(ApiKeyEntity apiKeyEntity) {
         return keyItem;
     }
 
+    public Application convert(ApplicationEntity applicationEntity) {
+        Application application = new Application();
+        application.setId(applicationEntity.getId());
+        application.setName(applicationEntity.getName());
+        application.setApiKeyMode(Application.ApiKeyModeEnum.valueOf(applicationEntity.getApiKeyMode().name()));
```

Comment:
I think the ApiKeyMode can be null in input of portal rest api.
Would it lead to a NPE here, accessing `applicationEntity.getApiKeyMode().name()` ?